ProcessLib/GroundwaterFlowFEM.h

```diff
@@ -34,11 +40,15 @@ class LocalAssemblerDataInterface
             Parameter<double, MeshLib::Element const&> const& hydraulic_conductivity,
             unsigned const integration_order) = 0;
 
-    virtual void assemble(std::vector<double> const& local_x,
-                          std::vector<double> const& local_x_prev_ts) = 0;
+    virtual void assemble(double const t, std::vector<double> const& local_x) = 0;
 
-    virtual void addToGlobal(GlobalMatrix& A, GlobalVector& rhs,
-            AssemblerLib::LocalToGlobalIndexMap::RowColumnIndices const&) const = 0;
+    // TODO: Don't we waste quite some time by having this as a virtual method?
+    //       If we put a callback arg to the assemble() call, this funcitonality can
+    //       be provided by a non-virtual method.
+    //       Otherwise we could also pass local matrices in with the assemble() call.
+    //       Thereby we would not have to expose global matrices/vectors.
```

Comment:
Suggest to move such discussions into issues, because here in the code nobody will see it.